environment_setup/install_clion.sh

```diff
@@ -4,12 +4,56 @@ echo "================================================================"
 echo "Installing CLion"
 echo "================================================================"
 
-# Clion can be installed through snapcraft
-# https://snapcraft.io/clion
-sudo apt-get install snapcraft
-sudo snap install clion --classic
+clion_version="2019.2.2"
+usr_location="/usr/local/bin/clion"
+
+if [ -e ${usr_location} ]
+then
+        echo "================================================================"
+        echo "CLion is already installed"
+        echo "================================================================"
+else
+	# Download clion
+        wget -O ~/Downloads/CLion-${clion_version}.tar.gz "https://download-cf.jetbrains.com/cpp/CLion-${clion_version}.tar.gz"
+
+	# Unzip and symlink to usr location
+        sudo mkdir -p /opt/clion
```

Comment:
would suggestion `/opt/clion_${clion_version}` here for the case that we install multiple versions of CLion. Also probably want to check if the folder exists in the `if` at the top just in case.

Reply:
In this case we can just un pack directly into `/opt` since the top level directory of the zip is `clion-${clion_version}`